### ISSUE TITLE
Data not captured on Android 4.4.2

### ISSUE THREAD

**Issue by YusriKhairi** [ahnada/PADListener#1]
I have tried the fixed app on my Android 4.4.2, but it doesn't capture any data on both Auto and Manual capture.

Using IPTables returns Error 101, on side note.

**Comment by ahnada** [comment]
Do you get any error messages when using Auto or Manual?
Have you tried a completely clean install (uninstalling PadListener, removing the security certificate, then go through the installation steps)?

If still not solved, please provide an adb log from your device while you launch PAD from PADListener and click through the main screen.

**Comment by YusriKhairi** [comment]
Nope, no errors on both. It just proceeds into the game without capturing any data.

**Comment by mizzpenny** [comment]
I am seeing the same thing on rooted Android 4.4.4 (Galaxy S5).. 

Uninstalled old Padlistener, downloaded the Fixed App from Wordpress page, export CA (tried both as "VPN and App" and "wifi", where I deleted the trusted cert before installing the other), tried all 3 listener modes.. and tried doing auto and manual capture..

I load PAD, it loads normally, but no data was captured.

Would be happy to send you adb log if you tell me how to generate the log :)

**Comment by ahnada** [comment]
Sorry folks, I've been unable to reproduce the problem and don't have a semi-recent Samsung device to develop on. Sadly I don't encounter any problems on my Galaxy S3.

The current plan is to enable more detailed logging and have people try it. Will post again when I have something.

**Comment by YusriKhairi** [comment]
I don't know if I did the right thing, but here's the log I captured from Logcat app.
[logcat-1706118941.txt](https://github.com/ahnada/PADListener/files/452050/logcat-1706118941.txt)
